scripts/manage/restore_backup_to_cloud_shell.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+~/spinnaker-for-gcp/scripts/manage/check_git_config.sh || exit 1
+
+while getopts ":p:r:h:" options; do
+  case $options in
+    p ) PROJECT_ID=$OPTARG ;;
+    r ) CONFIG_CSR_REPO=$OPTARG ;;
+    h ) GIT_HASH=$OPTARG ;;
+    \? ) bold "invalid option supplied: -$OPTARG"
+  esac
+done
+
+EXAMPLE_COMMAND="'restore_backup_to_cloud_shell.sh -p PROJECT -r REPOSITORY_NAME -h GIT_HASH'"
+
+if [ -z "$PROJECT_ID" ]; then
+  bold "Project id is required. $EXAMPLE_COMMAND"
+  exit 1
+fi
+
+if [ -z "$CONFIG_CSR_REPO" ]; then
+  bold "Cloud Source repository name is required. $EXAMPLE_COMMAND"
```

Comment:
We have `Cloud Source repository` here, but `Cloud source repository` down below. I'd go with `Cloud Source Repository` for both.